x-pack/plugins/apm/scripts/bootstrap_apm/README.md

```diff
@@ -0,0 +1,56 @@
+# Getting started
+
+A script for creating an elastic stack deployment on a remote cluster using `ecctl` and generating APM data using [apm-synthtrace](https://github.com/elastic/kibana/tree/main/packages/elastic-apm-synthtrace) tool.
+
+## Prerequisites for running the script
+
+- [ecctl](https://www.elastic.co/downloads/ecctl) installed
+- API key
+  - You can generate a key on [staging](https://staging.found.no/deployment-features/keys)
+- Make sure you have installed Kibana dependencies
+
+## Configuration
+
+Update the following configuration with your own credentials and store it in your home directory `$HOME/.ecctl/config.json`
+
+```
+$HOME/.ecctl/config.json
+
+{
+  "host": "https://staging.found.no",
+  "api_key": "API_KEY",
+  "region": "us-west-2",
```

Comment:
Should we document an EU region as default?

Reply:
generated by `ecctl init` but as far I can see staging has only gcp-us-central1 region available. Updated the template